Binary files pngnq-1.1.orig/._AUTHORS and pngnq-1.1/._AUTHORS differ
Binary files pngnq-1.1.orig/._ChangeLog and pngnq-1.1/._ChangeLog differ
Binary files pngnq-1.1.orig/._INSTALL and pngnq-1.1/._INSTALL differ
Binary files pngnq-1.1.orig/._Makefile.am and pngnq-1.1/._Makefile.am differ
Binary files pngnq-1.1.orig/._README and pngnq-1.1/._README differ
Binary files pngnq-1.1.orig/._configure and pngnq-1.1/._configure differ
Binary files pngnq-1.1.orig/._configure.ac and pngnq-1.1/._configure.ac differ
Binary files pngnq-1.1.orig/._pngnq.1 and pngnq-1.1/._pngnq.1 differ
Binary files pngnq-1.1.orig/freegetopt/._Makefile.am and pngnq-1.1/freegetopt/._Makefile.am differ
Binary files pngnq-1.1.orig/freegetopt/._README and pngnq-1.1/freegetopt/._README differ
Binary files pngnq-1.1.orig/src/._Makefile.am and pngnq-1.1/src/._Makefile.am differ
Binary files pngnq-1.1.orig/src/._colorspace.c and pngnq-1.1/src/._colorspace.c differ
Binary files pngnq-1.1.orig/src/._config.h.in and pngnq-1.1/src/._config.h.in differ
Binary files pngnq-1.1.orig/src/._errors.h and pngnq-1.1/src/._errors.h differ
Binary files pngnq-1.1.orig/src/._neuquant32.c and pngnq-1.1/src/._neuquant32.c differ
Binary files pngnq-1.1.orig/src/._neuquant32.h and pngnq-1.1/src/._neuquant32.h differ
Binary files pngnq-1.1.orig/src/._pngcomp.c and pngnq-1.1/src/._pngcomp.c differ
Binary files pngnq-1.1.orig/src/._pngnq.c and pngnq-1.1/src/._pngnq.c differ
Binary files pngnq-1.1.orig/src/._rwpng.h and pngnq-1.1/src/._rwpng.h differ
diff -Nurd pngnq-1.1.orig/src/rwpng.c pngnq-1.1/src/rwpng.c
--- pngnq-1.1.orig/src/rwpng.c	2010-05-10 18:42:56.000000000 -0400
+++ pngnq-1.1/src/rwpng.c	2025-01-28 01:56:14.000000000 -0500
@@ -31,6 +31,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 
 #include "png.h"        /* libpng header; includes zlib.h */
 #include "rwpng.h"      /* typedefs, common macros, public prototypes */
